build.gradle

```diff
@@ -94,20 +94,15 @@ dependencyRules {
     })
 
     // ethereum modules that need to be fixed. At least stop new dependencies being added
-    register(":ethereum:statetransition") {
-      allowed = [
-          ":infrastructure:",
-          ":ethereum:",
-          ":storage"
-      ]
-    }
-    register(":ethereum:weaksubjectivity") {
-      allowed = [
-          ":infrastructure:",
-          ":ethereum:",
-          ":storage"
-      ]
-    }
+    ['statetransition', 'executionlayer', 'weaksubjectivity'].forEach( {
```

Comment:
@ajsutton I added 'executionlayer' in the list of the bad guys since I needed to use the `InMemoryStorageSystemBuilder` for unit testing. Any concerns?